cmd/describe/environment.go

```diff
@@ -0,0 +1,164 @@
+package describe
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/dream11/odin/internal/service"
+	"github.com/dream11/odin/pkg/constant"
+	"github.com/dream11/odin/pkg/table"
+	environment "github.com/dream11/odin/proto/gen/go/dream11/od/environment/v1"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var name string
+
+
+var environmentClient = service.Environment{}
+var environmentCmd = &cobra.Command{
+	Use:   "env",
+	Short: "Describe environments",
+	Args: func(cmd *cobra.Command, args []string) error {
+		return cobra.NoArgs(cmd, args)
+	},
+	Long: `Describe  environment details`,
+	Run: func(cmd *cobra.Command, args []string) {
+		executeEnv(cmd)
+	},
+}
+
+func init() {
+	environmentCmd.Flags().StringVar(&name, "name", "", "name of the environment")
+	environmentCmd.Flags().StringVar(&serviceName, "service", "", "service deployed in this environment")
+	environmentCmd.Flags().StringVar(&component, "component", "", "component of the service")
```

Comment:
should we add `deployed in this environment` at the end to be consistent with the service description?